Modules/System/Azure Blob Services API/src/ABSClientImpl.Codeunit.al

```diff
@@ -590,10 +608,14 @@ codeunit 9051 "ABS Client Impl."
     var
         ABSOperationResponse: Codeunit "ABS Operation Response";
         Operation: Enum "ABS Operation";
-
     begin
         ABSOperationPayload.SetOperation(Operation::CopyBlob);
         ABSOperationPayload.SetOptionalParameters(ABSOptionalParameters);
+        if not SourceName.StartsWith('https://') then begin // SourceName is in the same storage account, use ABSOperationPayload to construct URI.
```

Comment:
If someone uses azurite (http://127.0.0.1) for testing, is this check here still valid?
Should we maybe update the documentation of the CopyBlob to highlight that we can use a fullname (URL to the source name only the blob name?

Reply:
Good point. A check for http:// is added.